packages/styles/scss/components/data-table/expandable/_data-table-expandable.scss

```
@@ -101,6 +101,8 @@
     td
     .#{$prefix}--child-row-inner-container {
     max-block-size: 100%;
+    padding-block: 1rem;
+    padding-block-end: 1.5rem;
```

We can just combine these in the `padding-block` definition 👍🏻 

```suggestion
    padding-block: 1rem 1.5rem;
```

---

@riddhybansal You can commit the suggestion directly in the "Commit suggestion" or push a new commit to address the changes TJ posted it